addons/project/data/project_demo.xml

```diff
@@ -863,68 +1101,100 @@
             <value eval="{'stage_id': ref('project_personal_stage_admin_5')}"/>
         </function>
 
-        <record id="message_task_1" model="mail.message">
-            <field name="model">project.task</field>
-            <field name="res_id" ref="project_task_22"/>
-            <field name="body">Hello Demo,
-There is a change in customer requirement.
-Can you check the document from customer again.
-Thanks,</field>
-            <field name="message_type">comment</field>
-            <field name="author_id" ref="base.partner_root"/>
-        </record>
-        <record id="message_task_2" model="mail.message">
-            <field name="model">project.task</field>
-            <field name="res_id" ref="project_task_22"/>
-            <field name="parent_id" ref="message_task_1"/>
-            <field name="body">Ok, I have checked the mail,
-I will update the document and let you know.</field>
-            <field name="message_type">comment</field>
-            <field name="author_id" ref="base.partner_demo"/>
-        </record>
-        <record id="message_task_3" model="mail.message">
-            <field name="model">project.task</field>
-            <field name="res_id" ref="project_task_22"/>
-            <field name="parent_id" ref="message_task_2"/>
-            <field name="body">Fine!
-Send it ASAP, its urgent.</field>
-            <field name="message_type">comment</field>
-            <field name="author_id" ref="base.partner_root"/>
```

Comment:
I moved it to a task where I thought it would make the demo data better and more consistent. It was previously in the chatter of the task 'Customer Analysis + Architecture', which is the very first task of the project that all other tasks depend on. It seemed a better fit for 'Panning and buddget', which is a few tasks down in the dependency graph